### PR TITLE
When the argument is a file with path provided, show the relative path of the file

### PR DESCRIPTION
**Pull request by ayushpoddar**
### Description

In order to demonstrate what this PR does, I am displaying the difference in the output of the command `colorls exe/colorls` (which is a file) before and after this PR

#### Before this PR
image

#### After this PR
image

#### Comment
This PR takes this gem one step closer to the behaviour of the in-built `ls` command.

### Details

- Relevant Issues : #524 
- Relevant PRs : (none)
- Type of change :
  - [ ] New feature
  - [x] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [ ] Addition or Improvement of documentation

### CODE REVIEW

File: lib/colorls/version.rb
Comment (avdv) [inline lib/colorls/version.rb:4]: We didn't release version 1.5.0 yet, so no need to bump the version.

File: spec/color_ls/flags_spec.rb
Comment (avdv) [inline spec/color_ls/flags_spec.rb:14]: Nice :+1:

**Review by avdv — CHANGES_REQUESTED** [review]
LGTM, but no need to bump the version.

File: lib/colorls/version.rb
Comment (ayushpoddar) [reply lib/colorls/version.rb]: Fixed

**Review by avdv — APPROVED** [review]

**Comment by avdv** [comment]
Thank you!

File: lib/colorls/version.rb
Comment (ayushpoddar) [reply lib/colorls/version.rb]: @avdv Do you need help in fixing the release script? By not releasing these latest changes, we are robbing people of all the new features being created

Comment (avdv) [reply lib/colorls/version.rb]: Thank you, but I already created a PR which fixes the problem. But I cannot merge it without an approval from another maintainer.

**Pull request merged**
